main/src/com/google/refine/operations/cell/MultiValuedCellSplitOperation.java

```diff
@@ -94,6 +95,11 @@
         _separator = separator;
         _mode = "separator";
         _regex = regex;
+        if (_regex) {
+            _pattern = Pattern.compile(_separator, Pattern.UNICODE_CHARACTER_CLASS);
```

Comment:
Obviously this problem exists in the current code, but I'm guessing it got flagged because it got moved/modified.

I'm not terribly worried about users launching DOS attacks against themselves.